docs/Concepts/Permissioning/PermissionsOverview.md

```diff
@@ -2,9 +2,8 @@
 
 GoQuorum supports two network permissioning models.
 
-* [Basic network permissioning](BasicNetworkPermissions.md)
-    Controls which nodes can connect to a given node and also to which nodes the given node can dial out to.
+* [Basic network permissioning](BasicNetworkPermissions.md): Controls which nodes can connect to a given node and also to which nodes the given node can dial out to.
+* [Enhanced network permissioning](Enhanced/EnhancedPermissionsOverview.md): Caters for enterprise-level needs by having a **smart contract-based permissioning model**. This allows for significant flexibility to manage nodes, accounts, and account-level access controls. There are two versions of enhanced network permissioning as described below:
```

Comment:
maybe find a synonym for "Caters" that will not show up as food related as the first meaning in dictionaries, can be very confusing for non english readers (and for instance macOS look up only provides the first meaning. Perhaps something like "Meets enterprise level needs"?

Reply:
sure. updated.